proc_scripts/load_data.py

```diff
@@ -52,27 +90,137 @@ def proc_spincore_CPMG_v1(s, fl=None):
     t2_axis = linspace(0,acq_time_s,nPoints)
     s.setaxis('nScans',r_[0:nScans])
     s.chunk('t',['ph1','tE','t2'],[nPhaseSteps,nEchoes,-1])
-    # OK, so I made some changes and then realized that we need to assume that
-    # the pulse sequence correctly balances the evolution between 2*p90_s/pi
-    # (cavanagh chpt 3 this is the evolution during the 90 -- I'm not positive
-    # if my expression is correct or not -- please do check/change, and leave
-    # this comment in some form) and the center of the 180 pulse appropriately
     s.setaxis('tE', (1+r_[0:nEchoes])*twice_tau)
     s.setaxis('ph1',r_[0.,2.]/4)
     s.ft('t2', shift=True)
     if fl is not None:
         fl.next('raw data - chunking ft')
         fl.image(s)
     s.ft(['ph1'])
+    return s
+
+def proc_bruker_T1CPMG_v1(s,d12,fl=None):
+    logger.info(strm("pre-processing of T1CPMG acquired on Bruker"))
+    assert s.get_prop('acq')['L'][21] == 2, "phase cycle isn't correct!"
+    assert s.get_prop('acq')['L'][22] == 4, "phase cycle isn't correct!"
+    s.chunk('indirect',['indirect','ph1','ph2'],[-1,2,4])
+    s.setaxis('ph1',r_[0,2]/4).setaxis('ph2',r_[0:4]/4)
+    s.ft(['ph1','ph2'])
+    s.reorder(['ph1','ph2','indirect'])
+    if fl is not None:
+        fl.next('raw data(t2,coh)')
+        fl.image(s)
+    s.setaxis('indirect', s.get_prop('vd'))
+    s.reorder(['ph1','ph2','indirect','t2'])
+    if fl is not None:
+        fl.next('raw data with indirect set')
+        fl.image(s)
+    #{{{ determining SW from anavpt
+    anavpt_info = [j for j in s.get_prop('pulprog').split('\n') if 'anavpt' in j.lower()]
+    anavpt_re = re.compile(r'.*\banavpt *= *([0-9]+)')
+    anavpt_matches = (anavpt_re.match(j) for j in anavpt_info)
+    for m in anavpt_matches:
+        if m is not None:
+            anavpt = int(m.groups()[0])
+    actual_SW = 20e6/anavpt # JF: check that this is based on the manual's definition of anavpt
```

Comment:
did you check the definition of `anavpt` in the manual?  Is the clock 20 MHz?  Equivalently, it might talk about 50 ns per sampling period.

Reply:
this is still outstanding until you reply

Reply:
![image](https://user-images.githubusercontent.com/52217688/104954766-dea12000-5996-11eb-8159-afb7d62b0d0f.png)
I think this answers your question that yes the clock is 20MHz